sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_http_logging_policy.py

```diff
@@ -21,149 +21,50 @@
-            try:
-                logger = response.context["logger"]
-
-                if not logger.isEnabledFor(logging.INFO):
-                    return
-
-                multi_record = os.environ.get(HttpLoggingPolicy.MULTI_RECORD_LOG, False)
-                if multi_record:
-                    logger.info("Response status: %r", http_response.status_code)
-                    logger.info("Response status reason: %r", ir.reason)
-                    try:
-                        logger.info("Elapsed Time: %r", ir.elapsed)
-                    except AttributeError:
-                        logger.info("Elapsed Time: %r", None)
-                    if http_response.status_code >= 400:
-                        sm = ir.text
-                        sm.replace("true", "True")
-                        sm.replace("false", "False")
-                        temp_sm = ast.literal_eval(sm)
-                        temp_sm['message'] = temp_sm['message'].replace("\r", " ")
-                        logger.into("Response status error message: %r", temp_sm['message'])
-                    logger.info("Response headers:")
-                    for res_header, value in http_response.headers.items():
-                        logger.info("    %r: %r", res_header, value)
-                    return
-                log_string = "Response status: {}".format(http_response.status_code)
-                log_string += "\nResponse status reason: {}".format(ir.reason)
-                try:
-                    log_string += "\nElapsed Time: {}".format(ir.elapsed)
-                except AttributeError:
-                    log_string += "\nElapsed Time: {}".format(None)
-                if http_response.status_code >= 400:
-                    sm = ir.text
-                    sm.replace("true", "True")
-                    sm.replace("false", "False")
-                    temp_sm = ast.literal_eval(sm)
-                    temp_sm['message'] = temp_sm['message'].replace("\r", " ")
-                    log_string += "\nResponse status error message: {}".format(temp_sm['message'])
-                log_string += "\nResponse headers:"
-                for res_header, value in http_response.headers.items():
-                    log_string += "\n    '{}': '{}'".format(res_header, value)
-                logger.info(log_string)
-            except Exception as err:  # pylint: disable=broad-except
-                logger.warning("Failed to log response: %s", repr(err))
-        else:
-            super().on_response(request, response)
+            options = response.context.options
+            logger = request.context.setdefault("logger", options.pop("logger", self.logger))
+            logger.info("Elapsed time in seconds: {}".format(time.time() - request.context.get("start_time")))
+            if http_response.status_code >= 400:
+                logger.info("Response error message: %r", _format_error(http_response.text()))
```

Comment:
Probably not a bad idea to wrap this whole section in a `try/except` just to prevent a failure to log from crashing the app (e.g. `json.loads` raises a `DecodeError` on the response body for some reason)